pkg/core/namespace_test.go

```diff
@@ -646,7 +653,17 @@ func keys(keys ...string) func(dict map[string]string) bool {
 
 func urlPath(path string) func(url *url.URL) bool {
 	return func(url *url.URL) bool {
-		return url.Path == path
+		if runtime.GOOS == "windows" && len(url.Scheme) == 1 {
```

Comment:
@sbawaska is there something in go-ape for this? If not, should there be?

Reply:
There is, but is is private: https://github.com/pivotal/go-ape/blob/master/pkg/furl/read.go#L55

Reply:
I had to add some code to handle letters which seems to vary between `D:` vs `d:`, maybe depending on some OS setting or type of filesystem. Not sure why that is, but I had to check the case of the drive letter part of the expected path to be able to match it.